pkg/boot/universalpayload/utilities.go

```diff
@@ -174,6 +174,52 @@ type FbFixScreenInfo struct {
 	Reserved     [2]uint16
 }
 
+type MCFGBaseAddressAllocation struct {
+	BaseAddr  EFIPhysicalAddress
+	PciSegGrp uint16
+	StartBus  uint8
+	EndBus    uint8
+	Reserved  uint32
+}
+
+type ResourceInfo struct {
+	Type        string
+	BaseAddress string
+	EndAddress  string
+	Attributes  string
+}
+
+type ResourceRegions struct {
+	Mmio64Base  uint64
+	Mmio64Limit uint64
+	Mmio32Base  uint64
+	Mmio32Limit uint64
+	IoPortBase  uint64
+	IoPortLimit uint64
+}
+
+const (
+	AcpiMcfgSysFilePath                 = "/sys/firmware/acpi/tables/MCFG"
+	AcpiMcfgPciSegInfoStructureSize     = 0xC
+	AcpiMcfgPciSegInfoDataLength        = 0xA
+	AcpiMcfgBaseAddressAllocationLenth  = 0x10
+	AcpiMcfgBaseAddressAllocationOffset = 0x2c
+	AcpiMcfgSignature                   = "MCFG"
+)
+
+const (
```

Comment:
The same initialism for this structure.

Reply:
I'm actually don't know how to apply initialism rule to Memory-mapped ConFiGuration..., but PCI should be in upper case.